src/views/PaymentNetwork.vue

```diff
@@ -74,54 +74,72 @@ function formatFiatCurrency(amount: number): string {
   return `${BASE_CURRENCY.symbol}${Math.round(amount).toLocaleString()}`;
 }
 
-function formatCryptoCurrency(amount: number, currency): string {
-  return `${amount.toLocaleString()} ${currency.code}`;
-}
-
 watch(paymentTx, () => {
   modalPostPaymentOpen.value = !!paymentTx.value;
 });
 </script>
 
 <template>
-  <h1>Network plan</h1>
-  <p class="text-md mb-4 leading-5">Pay to support your network on Snapshot</p>
+  <h1 class="leading-[44px] mb-[2px]">Network plan</h1>
+  <p class="mb-4 leading-[20px]">Pay to support your network on Snapshot</p>
 
   <form v-if="fxLoadStatus === 1" class="flex flex-col gap-4" @submit="pay">
     <fieldset>
-      <legend>
-        <h2 class="text-lg mb-2">Select your plan duration</h2>
+      <legend
+        class="text-[20px] mb-[12px] leading-none text-skin-heading font-semibold"
+      >
+        Select your plan duration
       </legend>
       <ol class="flex flex-col gap-2">
-        <li v-for="(plan, planId) in PLANS" :key="planId">
+        <li
+          v-for="(plan, planId) in PLANS"
+          :key="planId"
+          :class="[
+            {
+              'bg-[#384aff]/20 rounded-xl px-[1px] pb-[1px] mb-[-1px]':
+                plan.discount
+            }
+          ]"
+        >
+          <div v-if="plan.discount" class="p-2 text-center text-skin-primary">
+            Save
+            <BasePill class="py-1 !bg-skin-primary"
+              >{{ Math.floor(plan.discount) }}%</BasePill
+            >
+          </div>
           <label
-            class="flex gap-3 border border-skin-border hover:border-skin-text rounded-xl p-3 cursor-pointer"
+            class="flex relative bg-skin-bg gap-3 ring-1 ring-skin-border hover:ring-skin-text rounded-xl py-[12px] px-3 cursor-pointer"
```

Comment:
Can we move remove the hover effect here and on currency selector?

Reply:
You mean the border color change on hover ?

Reply:
Just no hover effect

Reply:
No hover effect on a clickable element ?

Seems inconsistent with other similar component, such as Spaces strategies/voting type selection

Reply:
Ok